app/controllers/profile_preview_card_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ProfilePreviewCardController < ApplicationController
+  layout false
+
+  def show
+    @actor = User.find_by(id: params[:userid])
+    @preview_id = params[:preview_card_id]
```

Comment:
BTW what is the preview id used for?

Reply:
The preview card needs to have a unique ID for the dropdown accessibility to work / to generate the HTML. It's a little odd, I know 🙁

Reply:
Can it be autogenerated?

Reply:
Hmmm... The only problem would be that the `buildCommentHTML` would need to know what the ID was in advance so that it could be added as `aria-controls="{the_id_of_the_dropdown}` to the trigger button... but this function is only called after `buildCommentHTML` has run and been inserted in the DOM 🤔

Reply:
Maybe it could be autogenerated if I refactored how we initialize these dropdowns - currently we rely on the `aria-controls` attribute being present, but maybe I'd be able to do something more like "find the closest sibling element that's a dropdown". That way we remove the reliance on the `aria-controls` being available upfront